src/Illuminate/Foundation/Console/GenerateLinksCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Console\Command;
+
+class GenerateLinksCommand extends Command
+{
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'links:generate';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create links';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        if ($links = $this->laravel['config']['filesystems.links']) {
+            foreach ($links as $link => $target) {
+                if (file_exists($link)) {
+                    $this->error("The [$link] directory already exists.");
```

Comment:
This could be confusing, if the location is not a directory. Perhaps do an is_file check to provide a more useful error message? That is, to tell the user if there is a file or a directory already there.

Reply:
Sorry, I'm a little confused. This is checking for the existence of the link, not the target. Would something like "The [$link] link already exists." make more sense?